### PR TITLE
Clarify instructions for "Keeping a consistent key"

### PR DESCRIPTION
**Pull request by AmySteam**
Changes proposed in this pull request:

- Clarify instructions for "Keeping a consistent ID" partial

[Preview page](https://deploy-preview-2274--developer-chrome-com.netlify.app/docs/webstore/identify_user#keep-consistent-id)

### CODE REVIEW

File: site/_includes/partials/extensions/reusing-prod-extension-id.md [hunk 1]
```diff
@@ -8,15 +8,22 @@ Package the extension directory into a `.zip` file and upload it to the [Chrome
 Dashboard](https://chrome.google.com/webstore/developer/dashboard) without publishing it:
 
 1.  On the Developer Dashboard, click **Add new item**.
-2.  Click **Browse files**, select the `.zip` extension, and upload it.
+2.  Click **Browse files**, select the extension `zip` file, and upload it.
```
Comment (dotproto) [inline site/_includes/partials/extensions/reusing-prod-extension-id.md:11]:
```suggestion
2.  Click **Browse files**, select the extension's `zip` file, and upload it.
```

File: site/_includes/partials/extensions/reusing-prod-extension-id.md [hunk 2]
```diff
@@ -8,15 +8,22 @@ Package the extension directory into a `.zip` file and upload it to the [Chrome
...
 3.  Go to the **Package** tab and click **View public key**.
 
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/8j12N4AyvHyjCZaFghv8.png", alt="Developer Dashboard
-Package tab", width="296", height="121" %}
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/8j12N4AyvHyjCZaFghv8.png", 
+alt="Developer Dashboard Package tab", width="296", height="121" %}
```
Comment (dotproto) [inline site/_includes/partials/extensions/reusing-prod-extension-id.md:15]:
```suggestion
alt="Developer Dashboard Package tab", width="296", height="121", class="screenshot" %}
```

Add the "screenshot" class to surround the image with an outline.

File: site/_includes/partials/extensions/reusing-prod-extension-id.md [hunk 3]
```diff
@@ -8,15 +8,22 @@ Package the extension directory into a `.zip` file and upload it to the [Chrome
...
-From the popup, copy the public key and add it to the manifest inside the unzipped directory under
-the [`"key"`]([/docs/extensions/mv3/manifest/key/) field. This way the extension will maintain the
-same ID.
+When the popup is open, follow these steps:
+
+1. Copy the code in between `-----BEGIN PUBLIC KEY-----` and `-----END PUBLIC KEY-----`.
+2. Remove the newlines in order to make it a single line of text.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/iOF372HS2DNkv5gLc1AA.png", alt="Public key popup",
+width="500", height="233" %}
```
Comment (dotproto) [inline site/_includes/partials/extensions/reusing-prod-extension-id.md:23]:
```suggestion
width="500", height="233", class="screenshot" %}
```

Add the "screenshot" class to surround the image with an outline.

File: site/_includes/partials/extensions/reusing-prod-extension-id.md [hunk 4]
```diff
@@ -28,11 +35,9 @@ same ID.
...
-Open the Extensions Management page at `chrome://extensions`, ensure developer mode is enabled and
-upload the unpackaged extension directory. Compare the extension ID on the extensions management
+Open the Extensions Management page at `chrome://extensions`, ensure **Developer mode** is enabled
+and upload the unpackaged extension directory. Compare the extension ID on the extensions management
 page to the Item ID in the Developer Dashboard. They should match.
 
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/AGorME3hdXd2YeKot5Nc.png", 
-alt="The ID of the
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/AGorME3hdXd2YeKot5Nc.png", alt="The ID of the
 extension match", width="356", height="352" %}
```
Comment (dotproto) [inline site/_includes/partials/extensions/reusing-prod-extension-id.md:43]:
```suggestion
extension match", width="356", height="352", class="screenshot" %}

```

Add the "screenshot" class to surround the image with an outline. Also, the file should end with a blank line. See [this SO answer](https://stackoverflow.com/a/729795/4959520) for more info.

**Review by dotproto — COMMENTED** [review]
Mostly nits. Once these are cleaned up I think we're ready to tag in Joe.

**Comment by AmySteam** [comment]
@jpmedley This PR is ready for an editorial review, thanks!

**Review by dotproto — APPROVED** [review]
LGTM

**Comment by dotproto** [comment]
@jpmedley, this is ready for you

File: site/_includes/partials/extensions/reusing-prod-extension-id.md [hunk 5]
```diff
@@ -8,15 +8,22 @@ Package the extension directory into a `.zip` file and upload it to the [Chrome
...
+When the popup is open, follow these steps:
+
+1. Copy the code in between `-----BEGIN PUBLIC KEY-----` and `-----END PUBLIC KEY-----`.
+2. Remove the newlines in order to make it a single line of text.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/iOF372HS2DNkv5gLc1AA.png", alt="Public key popup",
+width="500", height="233", class="screenshot" %}
+
+Add the code to the `manifest.json` under the [`"key"`]([/docs/extensions/mv3/manifest/key/) field.
+This way the extension will maintain the same ID.
```
Comment (jpmedley) [inline site/_includes/partials/extensions/reusing-prod-extension-id.md:26]:
```suggestion
This way the extension will use the same ID.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**